modules/gsp-cluster/main.tf

```
@@ -146,6 +146,7 @@ module "lambda_splunk_forwarder" {
   source = "../lambda_splunk_forwarder"
 
   enabled                   = "${local.enabled_addons["splunk"]}"
+  name                      = "kubernetes"
```

```suggestion
  name                      = "pods"
```

"kubernetes" seems a bit vague ... I imagine we'll have some kind of forwarder for "audit" logs in the future ... would these be named "kubernetes_audit" ?